sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -79,6 +79,32 @@ enum {
 
 #define SOF_SDW_MAX_DAI_NUM		8
 
+static const struct snd_soc_dapm_widget cs42l43_dmic_widgets[] = {
+	SND_SOC_DAPM_MIC("DMIC", NULL),
+};
+
+static const struct snd_soc_dapm_widget maxim_widgets[] = {
+	SND_SOC_DAPM_SPK("Left Spk", NULL),
+	SND_SOC_DAPM_SPK("Right Spk", NULL),
+};
```

Comment:
what is the purpose of this move? If these are codec specific then they should be remain where they were? Would this again increase data size?

Reply:
> what is the purpose of this move? If these are codec specific then they should be remain where they were? Would this again increase data size?

This commit is more like a preparation for next commit to add controls/widgets in sof_sdw_rtd_init()